Python/Vector Calculator/readme.md

```diff
@@ -0,0 +1,32 @@
+# Vector Calculator
+
+## Description
+
+Vector Calculator GUI based app built using Python that can fetch relation between 2 vectors along with the properties of each vector alone.
+
+Relations that can be calculated:
+
+* Dot product.
+* Cross product.
+* Angle between both vectors.
+* Component of each vector on the other.
+* Projection of each vector on the other.
+
+Properties that can be calculated:
+
+* Unit Direction of each vector.
+* Cosine Direction of each vector.
+* Cosine Angles of each vector.
+* Absolute Value of each vector. |A|
+
+## Tech Stack
+
+* Python3 -> Tkinter
+
+## Run Local
+
+`python3 calc.py`
+
```

Comment:
@XZANATOL add steps to run in local in these manner .
<img width="1080" alt="Screenshot 2022-03-03 at 1 01 42 PM" src="https://user-images.githubusercontent.com/78725162/156582225-e615590c-cbcb-4cf8-b21a-c84dc9403666.png">